packages/component-submission/client/components/PeoplePicker/PersonPod.js

```diff
@@ -113,20 +113,21 @@ class PersonPod extends React.Component {
     } = this.props
 
     let keywordList
+    /* eslint-disable react/no-array-index-key */
     const keywords = [].concat(focuses).concat(expertises)
```

Comment:
Can we make this something a bit nicer, like this:
```suggestion
    const keywords = [
      ...focuses,
      ...expertises
    
    ]
```